main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1734,7 +1729,7 @@ private void loadDescription(final String descriptionString, final IndexOutOfBou
 
                     fixRelativeLinks(descriptionView);
                     descriptionView.setMovementMethod(AnchorAwareLinkMovementMethod.getInstance());
-                    fixTextColor(descriptionString, descriptionView);
+                    fixTextColor(Html.toHtml(description), descriptionView);
```

Comment:
Can you add a comment that explains why not to use `descriptionString` (similar to the explanation in the commit message).

Reply:
Just found out that there is already proper span parsing in fixRelativeUrl method, so I thew away all HTML parsing with regexps and replaced it to neat span parsing. Sorry for several iterations